src/CTA.Rules.sln

```diff
@@ -1,7 +1,7 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio Version 16
-VisualStudioVersion = 16.0.30309.148
+# Visual Studio Version 17
```

Comment:
can I get some context here, I don't know why we updated this version. Curious to know :)

Reply:
Oh this was accidental :-) However not a bad idea to upgrade to latest :-)
Let me know if you have any reasons not to check this in.